File: javascript/day1/DayOneTask.js

	 let questions = 10;
	 let rightAnswer = 0; 
	 let countTime = System.currentTimeMillis();

	 let count = 0;
	

	  while(count < questions){
	  let first = (let)(Math.random()*10);
	  let second = (let)(Math.random()*10);
	   count++;


	  let store = first;
	  first = second;
	  second = store;

 	let prompt = require('prompt-sync')()
	console.log  prompt(input(first , second));
	 
		
	 if(first - second == answer){
	 console.log("You are right, good job! ");
	 rightAnswer++;
	 

	else if(first - second != answer)
	console.log prompt(input("You can try again: " + first + "-" +  second + ":"));
	

	if(first - second == answer){
	console.log("correct answer. Good job! ");
	rightAnswer++;
	
	}else{
	console.log("wrong result!");
	console.log("wrong! %n The right answer is: %d %s %d %s %d%n", first, "-", second, "=",(first - second));

        

	let stopTime = System.currentTimeMillis();
	 
	 console.log("The total number of your score is %d%s%d%n ", rightAnswer, "/", questions);	
	 let sumTime = countTime - stopTime ;
	 console.log("The time spent in seconds " + sumTime + "seconds");
	 

File: java/day 1/DayOneTask.java
import java.util.Scanner;

public class DayOneTask{
	public static void main(String [] args){
		
	 Scanner scanner = new Scanner(System.in);
	 int questions = 10;
	 int rightAnswer = 0; 
	 long countTime = System.currentTimeMillis();

	 int count = 0;
	

	  while(count < questions){
	  int first = (int)(Math.random()*10);
	  int second = (int)(Math.random()*10);
	   count++;


	  int store = first;
	  first = second;
	  second = store;
 	 
	 System.out.printf("What is %d %s %d %s",first, " - " , second , "=");
	 int answer = scanner.nextInt();
		
	 if(first - second == answer){
	 System.out.println("You are right, good job! ");
	 rightAnswer++;
	 }

	else if(first - second != answer){
	System.out.print("You can try again: " + first + "-" +  second + ":");
	answer = scanner.nextInt();

	if(first - second == answer){
	System.out.println("correct answer. Good job! ");
	rightAnswer++;
	
	}else{
	System.out.println("wrong result!");
	System.out.printf("wrong! %n The right answer is: %d %s %d %s %d%n", first, "-", second, "=",(first - second));

        }
             	
	}
	 
	  
		 
	 }
		long stopTime = System.currentTimeMillis();
	 
	 System.out.printf("The total number of your score is %d%s%d%n ", rightAnswer, "/", questions);	
	 long sumTime = countTime - stopTime ;
	 System.out.print("The time spent in seconds " + sumTime + "seconds");
	 

 }
}
